java/arcs/core/data/proto/policy.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package arcs;
+
+option java_package = "arcs.core.data.proto";
+option java_multiple_files = true;
+
+// Defines a data usage policy.
+message PolicyProto {
+  // Policy name.
+  string name = 1;
+
+  // Targets governed by the policy.
+  repeated PolicyTargetProto targets = 2;
+
+  // Additional config metadata for the policy.
+  repeated PolicyConfigProto configs = 3;
+
+  // Human readable description for the policy.
+  string description = 4;
+
+  enum EgressType {
+    EGRESS_TYPE_UNSPECIFIED = 0;
+    LOGGING = 1;
```

Comment:
There was also talk of distinguishing between logging to the centralized server vs. local on-device logging.

Though in Android case in particular local logging can then be uploaded to the server in case an error happened, so maybe not worth distinguishing now... 🤔

Reply:
We can always come back and add in a new one if needed.